src/OrchardCore.Modules/OrchardCore.Users/Workflows/Activities/ValidateUserTask.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Localization;
+using OrchardCore.Workflows.Abstractions.Models;
+using OrchardCore.Workflows.Activities;
+using OrchardCore.Workflows.Models;
+
+namespace OrchardCore.Users.Workflows.Activities
+{
+    public class ValidateUserTask : TaskActivity
+    {
+        private readonly IHttpContextAccessor _httpContextAccessor;
+        private readonly UserManager<IUser> _userManager;
+
+        public ValidateUserTask(UserManager<IUser> userManager, IHttpContextAccessor httpContextAccessor, IStringLocalizer<ValidateUserTask> localizer)
+        {
+            _httpContextAccessor = httpContextAccessor;
+            _userManager = userManager;
+            T = localizer;
+        }
+
+        private IStringLocalizer T { get; }
```

Comment:
As we agreed before T should define as field, I already did a PR to fix it in all places

Reply:
https://github.com/lukaskabrt/PoExtractor#limitations 

@hishamco @agriffard I think setting T as a field will break the PoExtractor.

Reply:
We already submitted a PR for that, @agriffard please let me know if there's a bug, so I can fix it in PO Extractor

/cc @jtkech

Reply:
And we also defined that `IStringLocalizer` would be named `S` and `IHtmlLocalizer` `H`. Note: there are some other places where `IStringLocalizer` and `IHtmlLocalizer` are still named `T`.

Oups, we didn't think about `PoExtractor`, @jptissot is right.

Reply:
So shall I start a PR on PoExtractor to fix this issue?

Reply:
Hmm, sorry maybe better to re-introduce the properties because we now know why ;)

But with only the get accessor.

Otherwise i assume it would be harder for PoExtractor to retrieve the localizers from private fields.

But personally i'm open to any solution, maybe ask it to @sebastienros

Reply:
There's a limitation in PoExtractor to extract localizers, I can let the extractor tool use both ways, so it will not breaks

Coz if we introduce the get accessor again for localizer, it's hard to remove it later for any reason, coz it will be a break change

I will dig into the PoExtractor, also @sebastienros feedback is always welcome

Reply:
The fix is read, just few tests and I will push a PR, FYI I'm able to eliminate the `IStringLocalizer` limitation for PoExtractor [here](https://github.com/lukaskabrt/PoExtractor#limitations)

Reply:
After many tests I made, I realize that the `PoExtractor` doesn't care if it's property or a field, it treat them as strings, but this will leads to some issues for example
```csharp
var S = new MyCollection();
...
var value = S["Indexer"];
```

The `S["Indexer"]` will be in POT which is totally wrong.

It's fine for now, but it needs some enhancement to avoid such cases, a proper solution is to find all the references for `IStringLocalizer`, `IHtmlLocalizer`

So no PR required for now, @jptissot no worry to apply the convention that we made earlier

/cc @lukaskabrt